jdaviz/app.py

```diff
@@ -1796,11 +1797,18 @@ def _reparent_subsets(self, old_parent, new_parent=None):
                     # Translate bounds through WCS if needed
                     if (self.config == "imviz" and
                             self._jdaviz_helper.plugins["Orientation"].link_type == "WCS"):
+
+                        old_angle, _, old_flip = get_compass_info(old_parent.coords, (10, 10))[-3:]
+                        new_angle, _, new_flip = get_compass_info(new_parent.coords, (10, 10))[-3:]
+                        relative_angle = new_angle - old_angle
+                        if old_flip != new_flip:
+                            relative_angle += 180
```

Comment:
this only matters for asymmetric shapes right?  In which case, rotating by 180 isn't the same as flipping, but maybe is better than nothing? 🤷

Reply:
Hmm, I think you're right that this isn't the proper handling for the flipped case, thinking about it again I don't know what I thought a 180 degree rotation for a symmetric shape would do 😆 . I just tested going from E-left to E-right and deleting E-left, and the subset disappeared (which...implies something else is wrong). I'll do some more thinking about the best way to handle that case.